HTML/EN/plugins/Qobuz/settings/basic.html

```diff
@@ -40,6 +40,28 @@
 				<option [% prefs.showYearWithAlbum ? "" : "selected" %] value="0" >[% "NO" | string %]</option>
 			</select>
 		[% END %]
+		<hr style="opacity:0.4" class="sub-sep"/>
+		[% WRAPPER settingGroup title="PLUGIN_QOBUZ_USE_CLASSICAL_ENHANCEMENTS" desc="PLUGIN_QOBUZ_USE_CLASSICAL_ENHANCEMENTS_DESC" %]
+			<select name="pref_useClassicalEnhancements" id="useClassicalEnhancements">
+				<option [% prefs.useClassicalEnhancements ? "selected" : "" %] value="1" >[% "YES" | string %]</option>
+				<option [% prefs.useClassicalEnhancements ? "" : "selected" %] value="0" >[% "NO" | string %]</option>
+			</select>
+			<!--[% "PLUGIN_QOBUZ_USE_CLASSICAL_ENHANCEMENTS_DESC" | string %]
+			[% "PLUGIN_QOBUZ_CLASSICAL_GENRES" | string %]&nbsp;
+			<input type="text" class="stdedit" name="pref_classicalGenres" id="classicalGenres" value="[% prefs.classicalGenres %]" size="80">-->
```

Comment:
Please remove unused code. You can still leave it in your clone by only un-staging these lines (easily done with something like SourceTree) if you want them to be around for future reference.